lexical/tokens.go

```diff
@@ -26,6 +26,8 @@ const T_PARENTH_R = 23
 const T_IN = 24
 const T_ASC = 25
 const T_LIKE = 26
+const T_NOT = 27
+const T_NOT_OR_T_LIKE = 28
```

Comment:
Did your Idea actually was `T_IN_OR_T_LIKE`, isn't?  I'm not too comfortable seeing a token that is used just for error messages. I prefer to suggest just one of the options (T_NOT) and maybe we can discuss better in another thread how to improve the parser error to show more then one expected tokens. What do you think?

Reply:
Yeah I wasn't too happy about that either. I didn't want to create an entirely different SyntaxError function though, so I squeezed in that token instead. I think you're right now. I'll change it to just suggest T_NOT for now, and we can leave the issue of leaving better parser errors for another day.

Reply:
Just letting you know I pushed another commit with this fix :)